app/assets/stylesheets/common/components/buttons.scss

```diff
@@ -51,6 +51,10 @@
   &:active,
   &.btn-active {
     @include linear-gradient($hover-bg-color 0%, $bg-color 100%);
+    // swaps the gradient on active to let dark theme match the light theme
+    @media (prefers-color-scheme: dark) {
+      @include linear-gradient($bg-color 0%, $hover-bg-color 100%);
```

Comment:
This will have unexpected side effects for users in a dark OS on a site that has disabled dark mode, the theme will honor this specific rule, but no other dark scheme colors will be used. It's not a big deal, the active state is very quick and barely visible, but I would prefer not to include this, we should not rely on `prefers-color-scheme` in regular stylesheets and having this in core would encourage future code readers to use this pattern. 

This change also doesn't fit with the rest of the PR, which addresses custom vars in the published pages endpoint.